libcontainer/specconv/spec_linux.go

```diff
@@ -227,6 +227,7 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 		labels = append(labels, fmt.Sprintf("%s=%s", k, v))
 	}
 	config := &configs.Config{
+		Umask:           spec.Process.User.Umask,
```

Comment:
Apparently `spec.Process.User` might be `nil`

Reply:
As an aside, we really should do some top-level cleaning up of `spec` so we don't have to have all these bloody `!= nil` checks everywhere. Oh what I would give for `Option<T>` in Go...